src/Components/v2/ArtworkFilter/ArtworkFilterMobileActionSheet.tsx

```diff
@@ -1,36 +1,56 @@
-import { Box, Button, Sans, Spacer } from "@artsy/palette"
-import { MobileTopBar } from "Components/MobileTopBar"
+import { Box, Button, Sans, Flex, color } from "@artsy/palette"
 import React, { SFC } from "react"
 import styled from "styled-components"
-import { useArtworkFilterContext } from "./ArtworkFilterContext"
+import {
+  useArtworkFilterContext,
+  initialArtworkFilterState,
+} from "./ArtworkFilterContext"
+import { isEqual, omit } from "lodash"
 
 export const ArtworkFilterMobileActionSheet: SFC<{
   children: JSX.Element
   onClose: () => void
 }> = ({ children, onClose }) => {
   const filterContext = useArtworkFilterContext()
 
+  // TODO: `filterContext.hasFilters` seems to be what we want but is never actually `false`?
```

Comment:
Did you dig at all? It looks like it's set up with tests and everything, so I'm surprised that it's broken 😭 .

Reply:
I'm not 100% sure on it's meaning. Though here it's a bit more particular since we're only really talking about what's in the mobile UI. I'll dig a little bit and remove this comment.

Reply:
Here's an example where `hasFilters` is false: https://www.artsy.net/search?term=hhhhhhhhhhhhhhhh

It looks like that prop only applies to search pages [and the ZeroState](https://github.com/damassi/reaction/blob/917f59f79fac9af348e4944049b077a9e6aead86/src/Apps/Search/Components/ZeroState.tsx#L30) at the moment. (This ZeroState should be moved into the `Apps/Search/Components` folder at some point.)

Reply:
OK, makes sense. Updated comment just to be clear.